sql/connect/client/jdbc/src/main/scala/org/apache/spark/sql/connect/client/jdbc/SparkConnectStatement.scala

```diff
@@ -35,14 +35,21 @@ class SparkConnectStatement(conn: SparkConnectConnection) extends Statement {
   override def close(): Unit = synchronized {
     if (!closed) {
       if (operationId != null) {
-        conn.spark.interruptOperation(operationId)
+        try {
+          conn.spark.interruptOperation(operationId)
+        } catch {
+          case _: Exception =>
```

Comment:
Can this be more specific exception, @vinodkc ?

Reply:
~~Can we just use the utility method `Utils.closeQuietly`?~~

nvm, this is fine

Reply:
> Can we just use the utility method `Utils.closeQuietly`?

+1